orgs/prerelease.json

```diff
@@ -0,0 +1,12 @@
+{
+  "orgName": "HEDA - Prerelease Dev Workspace",
```

Comment:
@davisagli Not really a big deal, but in #768 I renamed the org configs to "EDA" instead of "HEDA" - can this just be "EDA" too?

Reply:
@robertbwatson good catch, I made that update.